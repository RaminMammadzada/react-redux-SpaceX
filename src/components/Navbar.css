.navbar {
  display: flex;
  justify-content: space-between;
  background-color: rgb(255, 203, 203);
  align-items: center;
  padding: 0 40px;
}

.nav-item {
  background-color: rgb(128, 47, 47);
  margin: 2px;
  color: aliceblue;
  text-decoration: none;
}

.nav-item:hover {
  background-color: rgb(145, 125, 125);
}
