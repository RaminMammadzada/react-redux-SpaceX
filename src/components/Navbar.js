import React from 'react';
import { Link } from 'react-router-dom';
import { Button } from 'react-bootstrap';
import './Navbar.css';
import 'bootstrap/dist/css/bootstrap.min.css';
import logo from '../images/spacex_logo.png';

const Navbar = () => (
  <div className="navbar sticky-top">
    <div>
      <img src={logo} alt="logo" />
    </div>
    <div className="nav-links">
      <Link className="mx-2" to="/missions">
        <Button variant="outline-light">Missions</Button>
      </Link>
      <Link className="mx-2" to="/about">
        <Button variant="outline-light">About</Button>
      </Link>
    </div>
  </div>
);

export default Navbar;
