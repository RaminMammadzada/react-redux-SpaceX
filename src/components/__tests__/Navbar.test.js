import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../store';
import Navbar from '../Navbar';

describe('Navbar component - ', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Provider store={store}>
          <Navbar />
        </Provider>
      </BrowserRouter>,
    );

    expect(component).toMatchSnapshot();
  });
});
