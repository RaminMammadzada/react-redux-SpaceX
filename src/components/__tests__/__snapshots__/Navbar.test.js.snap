// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component -  matches the snapshot 1`] = `
<div
  className="navbar sticky-top"
>
  <div>
    <img
      alt="logo"
      src="spacex_logo.png"
    />
  </div>
  <div
    className="nav-links"
  >
    <a
      className="mx-2"
      href="/missions"
      onClick={[Function]}
    >
      <button
        className="btn btn-outline-light"
        disabled={false}
        type="button"
      >
        Missions
      </button>
    </a>
    <a
      className="mx-2"
      href="/about"
      onClick={[Function]}
    >
      <button
        className="btn btn-outline-light"
        disabled={false}
        type="button"
      >
        About
      </button>
    </a>
  </div>
</div>
`;
